src/main.rs

```diff
@@ -17,18 +17,45 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
+use clap::{CommandFactory, ErrorKind, Parser};
+
 use astarte_device_sdk::options::AstarteOptions;
 use astarte_device_sdk::registration;
 use astarte_device_sdk::AstarteDeviceSdk;
 
 use astarte_message_hub::config::MessageHubOptions;
 use astarte_message_hub::error::AstarteMessageHubError;
 
+/// A central service that runs on (Linux) devices for collecting and delivering messages from N
+/// apps using 1 MQTT connection to Astarte.
+#[derive(Parser, Debug)]
+#[clap(author, version, about, long_about = None)]
+struct Args {
+    /// Path to a valid .toml file containing the message hub configuration.
+    /// Do not provide this option with the `store_directory` option.
+    #[clap(short, long, value_parser)]
```

Comment:
Is the `value_parser` necessary? I believe it can be elided and will be defaulted for those types.